lib/dialects/abstract/query-generator.js

```diff
@@ -2335,6 +2336,8 @@ const QueryGenerator = {
       case Op.between:
       case Op.notBetween:
         return this._joinKeyValue(key, `${this.escape(value[0])} AND ${this.escape(value[1])}`, comparator, options.prefix);
+      case Op.ts:
```

Comment:
Is this necessary? I think this really limits the use cases of this operator. https://www.postgresql.org/docs/current/static/functions-textsearch.html 

Maybe the same could be achieved using `sequelize.fn`:

```javascript
Post.findAll({
  where: {
    my_ts_vector: {
      [Op.ts]: sequielize.fn('plainto_tsquery', 'english', 'my ts query')
    }
  }
});
```

What do you think?

Reply:
@yrong Forked your fork (LOL). https://github.com/jonykalavera/sequelize/commit/bd2da2b23291df50333a5b8660c822777b3951cd

This allows me to build the ts query myself. Do you think we can get to a middle ground?